src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1708,71 +1708,128 @@ Below is an example for adding a certificate to a Java cacerts file:
+If the workflow with the same name is already exists in specified bucket in the catalog, it will be created the new revision of this workflow.
+We recommend to specify commit message for easier differentiation the versions of a workflow.
+
+==== Get a Workflow from the Catalog
+When the workflow is selected from catalog you have two options:
+`Open as a new Workflow`
+This option will open the workflow from catalog as new Workflow in Studio
+`Append to current Workflow`
+When the user choose this option, it's supposed that the user wants to append the workflow from catalog to workflow already opened inside studio. Please open the workflow to which you want to append before.
+
+=== Workflow Catalog Portal
+The Workflow Catalog Portal provides interfaces for easy administering the catalog. The most important of provided features will be described next.
+
+==== Creating the new buckets
+You need to press `plus` button from above of buckets. In opened window you need to specify Bucket name and associate with user group.
+For security reason there are groups associated to buckets. The user from same group can view the content of bucket.
+There is a special default group - `no group` in the dropdown list of window. The bucket associated with this default group can be viewed by any user.
+
+==== Removing workflows inside bucket
+For removing workflows, first the user should select the bucket containing the workflows. Then the user can select multiple workflows with pressed CTRL button. To delete the selected workflows push the `trash` button.
 
-Evaluation priority is respected through operator precedence (`AND` over `OR`), so the following statement:
+===== Importing/Exporting Workflows into the Catalog
+In the Workflow Catalog, it is possible to Export Workflows as an archive, then Import them.
+You can export Workflows into an archive, which can be transferred by any file transfer method (email, usb key, etc.) and then imported into the other Scheduler.
 
-    Search for all Workflows that can deploy an OpenStack instance
+To export the selected Workflows as a ZIP archive from Workflow catalog portal: select a bucket, then the targeted Workflows (you can select multiple workflows with pressed CTRL button). Press "Download Workflows" button from portal view. You'll get a ZIP file which can be transferred to the machine on which the other Scheduler runs.
+To import the zip archive of workflows to the Catalog from portal, select the bucket to which the Workflows will be added. Then press `Upload Workflows` button in portal and follow the instructions `Upload a ZIP archive of workflows` opened window.
 
-can be expressed as:
+==== List revisions of workflow
+In portal you always see the last revision of catalog. In order to see all revisions of selected workflow, please click on `clock` button inside description area.
```

Comment:
In portal you always see the last revision of catalog
=> The last revision of the Catalog is always displayed on the portal.